src/Controller/Gateways/Pix.php

```diff
@@ -69,19 +69,23 @@ public function field_pix_qrcode_expiration_time()
     public function field_pix_additional_data()
     {
         return [
-            'title'       => __('Additional information', 'woo-pagarme-payments'),
-            'description' => __('Set of key and value used to add information to the generated pix. This will be visible to the buyer during the payment process.', 'woo-pagarme-payments'),
-            'desc_tip'    => true,
+            'title' => __('Additional information', 'woo-pagarme-payments'),
+            'description' => __(
+                'Set of key and value used to add information to the generated pix.' +
```

Comment:
1. Missing space at the end of the string
2. Concatenate with . (dot)

Reply:
Fixed